src/packaging/pkgconfig/mpich.pc.in

```diff
@@ -9,7 +9,7 @@ Description: High Performance and portable MPI
 Version: @MPICH_VERSION@
 URL: http://www.mcs.anl.gov/research/projects/mpich
 Requires:
-Libs: @WRAPPER_RPATH_LDFLAGS@ @WRAPPER_LDFLAGS@ -L${libdir} -l@MPILIBNAME@ @LPMPILIBNAME@ @WRAPPER_LIBS@
+Libs: @WRAPPER_C_DYNAMIC_LOADING_FLAGS@ @WRAPPER_LDFLAGS@ -L${libdir} -l@MPILIBNAME@ @LPMPILIBNAME@ @WRAPPER_LIBS@
```

Comment:
What do the dynamic loading flags have to do with C? I'm unsure why this variable name went from referencing LDFLAGS to C.

Reply:
They are specific to a compiler (to the linker, actually).  Before this patch, we were using the flags from the C compiler in pkg-config.  This patch retains that part without any changes.  I just named it to refer to "C", because that's what it is even though the previous name was not calling it that.

Reply:
Fair enough. That squares with the comments in `configure.ac`, so it sounds OK to me. Signing off...

Reply:
No, I think I prefer the more verbose version. It's easier to understand at a glance. I've already signed off, just forgot to hit the merge button.